### PR TITLE
Issue #13 - Add EBS Storage Classes

### LINKED ISSUE #13: Add EBS Storage Classes
Let's shoot for 2 "classes" of storage. a "fast" and "slow"

### PR DESCRIPTION
**Pull request by themoosman**
Add three (fast, medium, slow) AWS EBS StorageClasses.  

Resolves #13

### CODE REVIEW

File: templates/aws-ebs-storage-classes.yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+      encrypted: "${ENCRYPT_STORAGE}"
+      type: gp2
+    provisioner: kubernetes.io/aws-ebs
+    reclaimPolicy: Delete
+    volumeBindingMode: Immediate
+  - kind: StorageClass
+    apiVersion: storage.k8s.io/v1
+    metadata:
+      annotations:
+        storageclass.beta.kubernetes.io/is-default-class: "false"
```
[the remark this answers is not in the document]
Comment (themoosman) [reply templates/aws-ebs-storage-classes.yaml]: You want the slow SC to be the default?

[the remark this answers is not in the document]
Comment (themoosman) [reply templates/aws-ebs-storage-classes.yaml]: @etsauer

[the remark this answers is not in the document]
Comment (etsauer) [reply templates/aws-ebs-storage-classes.yaml]: @themoosman yes. the assumption being that we would want users of this theoretical cluster to have to "opt-in" to consume more expensive storage.

[the remark this answers is not in the document]
Comment (themoosman) [reply templates/aws-ebs-storage-classes.yaml]: @etsauer Updated.

**Pull request merged**